src/Command/DetectCommand.php

```diff
@@ -93,7 +93,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         foreach ($paths as $path) {
-            $finder->in($path);
+            $finder->in($path)->followLinks();
```

Comment:
I guess it makes more sense to change this once globally in line 84 instead of calling it for each path in the foreach loop, right?